tests/h/traversal/organization_test.py

```diff
@@ -24,10 +24,10 @@ def test_it_404s_if_the_organization_doesnt_exist(
         with pytest.raises(KeyError):
             OrganizationRoot(pyramid_request)[sentinel.non_existent_pubid]
 
-    @pytest.fixture
+    @pytest.fixture(autouse=True)
     def with_noise_organization(self, factories):
         # Add a handful of organizations to the DB to make the test realistic.
-        factories.Organization.generate_batch(size=2)
+        factories.Organization.create_batch(size=2)
```

Comment:
This is a coverage fix, but actually this was a genuine mistake introduced in the previous PR. The fixture wasn't used and didn't work if you called it... great job... slow clap me.